README.md

```diff
@@ -11,15 +11,12 @@ THIS IS RESEARCH CODE PROVIDED TO YOU "AS IS" WITH NO WARRANTIES OF CORRECTNESS.
 ## Contents
 1. [Software Dependencies and Installation](#software-dependencies-and-installation)
 2. [Jupyter Notebooks](#jupyter-notebooks)
-   - [Downloading GUNW products using ariaDownload.py - ariaDownload program tutorial](https://nbviewer.jupyter.org/github/aria-tools/ARIA-tools-docs/blob/master/JupyterDocs/ariaDownload/ariaDownload_tutorial.ipynb)
-   - [Manipulating GUNW products using GDAL (python and commandline) - GDAL basics tutorial](https://nbviewer.jupyter.org/github/aria-tools/ARIA-tools-docs/blob/master/JupyterDocs/GDAL_basics/GDAL_basics.ipynb)
-   - [Manipulating Layers of ARIA standard GUNW products - ariaExtract.py program tutorial](https://nbviewer.jupyter.org/github/aria-tools/ARIA-tools-docs/blob/master/JupyterDocs/ariaExtract/ariaExtract_tutorial.ipynb)
-   - [Assessing data quality and spatiotemporal coverage of interferograms - ariaPlot.py program tutorial](https://nbviewer.jupyter.org/github/aria-tools/ARIA-tools-docs/blob/master/JupyterDocs/ariaPlot/ariaPlot_tutorial.ipynb)
-   - [Preparing ARIA standard GUNW products layers for time-series analysis using ariaTSsetup.py - ariaTSsetup.py program tutorial](https://nbviewer.jupyter.org/github/aria-tools/ARIA-tools-docs/blob/master/JupyterDocs/ariaTSsetup/ariaTSsetup_tutorial.ipynb)
-   - [Creating a new area of interest](https://nbviewer.jupyter.org/github/aria-tools/ARIA-tools-docs/blob/master/JupyterDocs/ariaAOIassist/ariaAOIassist_tutorial.ipynb)
-    - [Calculating phase triplet misclosures](https://nbviewer.jupyter.org/github/aria-tools/ARIA-tools-docs/blob/master/JupyterDocs/ariaMisclosure/ariaMisclosure_tutorial.ipynb)
-    - [InSAR Time Series Analysis using MintPy and ARIA products](https://nbviewer.jupyter.org/github/aria-tools/ARIA-tools-docs/blob/master/JupyterDocs/NISAR/L2_interseismic/mintpySF/smallbaselineApp_aria.ipynb)
-    - [External - Time-series InSAR processing using the Miami INsar Time-series software in PYthon tutorial](https://github.com/insarlab/MintPy)    
+   - [Downloading GUNW products using ariaDownload.py - ariaDownload program tutorial](#downloading-gunw-products-using-ariadownload.py---ariadownload-program-tutorial)
```

Comment:
@rzinke can you make this consistent with how we are doing it now. Also note that you are deleting some which wee added in last PR.

We decided from now on to runn all the cells of the notebooks and pre-view them for others such they have life documentation.